README.md

```diff
@@ -149,9 +168,9 @@ API to acknowledge the event, set `wait: true`.
 
 ### Prefix
 
-The ``prefix`` of the logs and metrics.
+The `prefix` of the logs and metrics.
 
-```yaml
+``yaml
```

Comment:
Aren't you missing a "`" sign on opening and closing?

Reply:
Good catch, missed this one. Fixed!